app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: [dict], cafe: Cafe) -> str:
```

Comment:
The function annotation for `friends` is not correct. In Python, you should use `List[dict]` instead of `[dict]` to indicate a list of dictionaries. Make sure to import `List` from `typing`.